src/bacnet/basic/object/iv.c

```diff
@@ -233,22 +233,65 @@ bool Integer_Value_Present_Value_Set(
 bool Integer_Value_Object_Name(
     uint32_t object_instance, BACNET_CHARACTER_STRING *object_name)
 {
+    static char text[32] = ""; /* okay for single thread */
```

Comment:
Since those object-name functions are copying the string to the character-string which already has storage, there is no longer a need to be static (I think this was pattern from previous examples of just c-string). I'll remove those 'static' strings. Thanks for the hint!